source/extensions/filters/http/ext_proc/matching_utils.cc

```diff
@@ -63,16 +63,35 @@ ExpressionManager::evaluateAttributes(const Filters::Common::Expr::Activation& a
       continue;
     }
 
+    // Possible types from source/extensions/filters/common/expr/context.cc:
+    // - String
+    // - StringView
+    // - Map
+    // - Timestamp
+    // - Int64
+    // - Duration
+    // - Bool
+    // - Uint64
+    // - Bytes
+    //
+    // Of these, we need to handle
+    // - bool as bool
+    // - int64, uint64 as number
+    // - everything else as string via print
+    //
+    // Handling all value types here would be graceful but is not currently
+    // testable and drives down coverage %. This is not a _great_ reason to
+    // not do it; will get feedback from reviewers.
```

Comment:
As long as not handling has a graceful fallback which it does this seems fine to me.